Makefile

```diff
@@ -13,11 +14,13 @@ DEVELOPMENT?=false
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 AWS_REGION?=us-west-2
 IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+export KOPS_CLUSTER_NAME=$(shell cat /dev/urandom | LC_ALL=C tr -dc '[:alpha:]' | tr '[:upper:]' '[:lower:]' | head -c 32).prod.test.pdx.kops-ci.model-rocket.aws.dev
```

Comment:
Delete this  and put it in the create_store_name script most likely

Reply:
If we put it there, we can generate a store name and cluster name the same way using the pull sha1

Reply:
Yup that's what I want to do, just doing some testing before I address that